src/main.cpp

```diff
@@ -374,13 +430,14 @@ Frame::Frame()
 }
 
 bool App::OnInit() {
-    Frame* frame = new Frame();
+    // Do NOT delete this. frame->Show() makes it so that the library takes care
+    // of delete’ing the frame when appropriate.
+    auto* frame = new Frame();
     frame->Show(true);
     return true;
 }
 
-void Frame::OnExit(wxCommandEvent& event) {
-    (void)event;
+void Frame::OnExit(wxCommandEvent&) {
```

Comment:
Casting to `void` is one way of doing is, but not naming the parameter is simpler imo.